README.md

```diff
@@ -1,15 +1,36 @@
-# Assistive Technology ARIA Experience Assessment Project
+# ARIA and Assistive Technologies (ARIA-AT)
 
-The aim of this project is to facilitate consistently effective rendering of ARIA-enabled UIs by assistive technologies by:
+This repo contains a low-level test suite and harness for assessing the interoperability of AT interpretations of ARIA. These tests are designed to be modular and embeddable in higher-level applications.
```

Comment:
Instead of "for assessing the interoperability of AT interpretations of ARIA",
I suggest "for assessing AT support of ARIA".

That would be a bit simpler.